packages/headless/contributors/adding-a-sub-package.md

```diff
@@ -8,7 +8,7 @@ Headless provides exports through multiple sub packages. A sub-package groups to
 1. Create an entry file for your sub-package inside the `src` directory (e.g. case-assist.ts).
 2. Configure nodejs and browser bundles inside `rollup.config.js` for your entry file.
 3. Create a new directory with the name of your sub-package at the project root.
-4. Inside the new directory, add a `package.json` file. Add the paths to your bundled files and type definitions. Mark the package `private` to prevent `lerna` from bumping it's version.
```

Comment:
Don't we still need to set private if we don't want the pkg to be published?

Reply:
Overall, if a package is not meant to be published, it's good practice to mark it as such